package.json

```diff
@@ -22,7 +22,16 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
+    "build-with-docker": "docker run  --rm -it -v $(pwd):/jetpack -w /jetpack --name jetpack-builder oskosk/jetpack-builder bash -c 'yarn build'",
```

Comment:
I'd suggest moving this to a docker-compose file, which is more cross-platform compatible. (`$(pwd)` doesn't work on Windows). Then you can just do `docker-compose -f docker-builder.yml up`.